### PR TITLE
fixed some grammatical mistakes in Readme.md

### LINKED ISSUE #1: Grammatical fixes in documentation
Fixes for grammatical errors are welcomed through PRs. Also, if you certain easy and quick ReactJS project ideas, you can add them to the README of this repo.

### PR DESCRIPTION
**Pull request by vineetnilawar**
fixed some grammatical mistakes in Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # All About ReactJS
 
-### If you want to contribute to this project, consider reading the [contribution.md](https://github.com/DemonDaddy22/all-about-reactJS/blob/master/contribution.md) first to get a better idea on how to contribute to this repo.
+### If you want to contribute to this project, consider reading the [contribution.md](https://github.com/DemonDaddy22/all-about-reactJS/blob/master/contribution.md) first to get a better idea of how to contribute to this repo.
```
Comment (DemonDaddy22) [inline README.md:3]: Did you use any tool for this change, because I think 'on' makes more sense than 'of'?

Comment (vineetnilawar) [reply README.md:3]: yeah, I checked it on Grammarly too

Comment (DemonDaddy22) [reply README.md:3]: Okay, cool then.

**Review by DemonDaddy22 — APPROVED** [review]

**Comment by DemonDaddy22** [comment]
@all-contributors please add @vineetnilawar for documentation

**Pull request merged**